requirements.txt

```diff
@@ -10,16 +10,43 @@
 # `Pipfile.lock` and then regenerate `requirements*.txt`.
 ################################################################################
 
-coverage
-coveralls
-dj-database-url
-django
-django-cors-headers
-django-extensions
-django-heroku
-djangorestframework
-gunicorn
-jwt
-pipenv-to-requirements
-psycopg2-binary
-python-decouple
+asn1crypto==0.24.0
```

Comment:
@felixkiryowa please use `pipenv run pipenv_to_requirements` to generate the `requirements.txt` file